# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../libtai/libtai-0.60-0001-include-stdlib.h.patch
# Copyright (C) 2011 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

diff --git a/check.c b/check.c
index 9c7503c..b0d1484 100644
--- a/check.c
+++ b/check.c
@@ -1,4 +1,5 @@
 #include <stdio.h>
+#include <stdlib.h> /* exit */
 #include <time.h>
 #include "tai.h"
 #include "leapsecs.h"
diff --git a/easter.c b/easter.c
index 29355f4..d28f9e6 100644
--- a/easter.c
+++ b/easter.c
@@ -1,4 +1,5 @@
 #include <stdio.h>
+#include <stdlib.h> /* exit */
 #include "caldate.h"
 
 char *dayname[7] = { "Sun", "Mon", "Tue", "Wed", "Thu", "Fri", "Sat" } ;
diff --git a/leapsecs.c b/leapsecs.c
index e5db267..d450c56 100644
--- a/leapsecs.c
+++ b/leapsecs.c
@@ -1,4 +1,5 @@
 #include <stdio.h>
+#include <stdlib.h> /* exit */
 #include "tai.h"
 #include "leapsecs.h"
 #include "caldate.h"
diff --git a/leapsecs_read.c b/leapsecs_read.c
index 62c8712..3919f35 100644
--- a/leapsecs_read.c
+++ b/leapsecs_read.c
@@ -1,6 +1,7 @@
 #include <sys/types.h>
 #include <sys/stat.h>
 #include <fcntl.h>
+#include <stdlib.h> /* free, malloc */
 #include <errno.h>
 extern int errno;
 #include "tai.h"
diff --git a/nowutc.c b/nowutc.c
index c327d75..9d13223 100644
--- a/nowutc.c
+++ b/nowutc.c
@@ -1,4 +1,5 @@
 #include <stdio.h>
+#include <stdlib.h> /* exit */
 #include "leapsecs.h"
 #include "tai.h"
 #include "taia.h"
diff --git a/yearcal.c b/yearcal.c
index 545e81e..a454039 100644
--- a/yearcal.c
+++ b/yearcal.c
@@ -1,4 +1,5 @@
 #include <stdio.h>
+#include <stdlib.h> /* exit */
 #include "caldate.h"
 
 char *montab[] = {
